chapter_preliminaries/calculus_vn.md

```diff
@@ -400,28 +400,30 @@ $$\frac{\partial y}{\partial x_i} = \frac{\partial f}{\partial x_i} = f_{x_i} =
 ## Gradients
 -->
 
-## *dịch tiêu đề phía trên*
+## Gradient
 
 <!--
 We can concatenate partial derivatives of a multivariate function with respect to all its variables to obtain the *gradient* vector of the function.
 Suppose that the input of function $f: \mathbb{R}^n \rightarrow \mathbb{R}$ is an $n$-dimensional vector $\mathbf{x} = [x_1, x_2, \ldots, x_n]^\top$ and the output is a scalar. The gradient of the function $f(\mathbf{x})$ with respect to $\mathbf{x}$ is a vector of $n$ partial derivatives:
 -->
 
-*dịch đoạn phía trên*
+Chúng ta có thể ghép các đạo hàm riêng của mọi biến trong một hàm đa biến để thu được vector *gradient* của hàm số đó.
```

Comment:
```suggestion
Chúng ta có thể ghép các đạo hàm riêng theo tất cả các biến trong một hàm đa biến để thu được vector *gradient* của hàm số đó.
```
Mình thường dịch `đạo hàm theo abcd...`.

Reply:
`with respect to` mình thấy cũng nên dịch là `theo`